source/includes/reference/_webhook_events.md

```diff
@@ -67,10 +85,14 @@ public boolean verifySignature(String encodedPublicKey, String signature, String
 
 Each outgoing webhook request is signed.
 Although event payloads do not contain any sensitive information, you may want to
-verify if the request is coming from TransferWise (however this is optional).
-We advise you not to process any requests where signature appears to be forged.
+verify if the request is coming from TransferWise and that it has not been forged or tampered with.
+You should not process any requests with signatures that fail verification.
+
+Signatures are generated using an RSA key and SHA256 digest of the message body.
+
+Signatures are transmitted using the `X-Signature-SHA256` request header. Signatures are Base64 encoded.
```

Comment:
three sentences starting with `Signatures are ...`. maybe it's perfectly normal but just checking if there's a better way to communicate :)

Reply:
Yup, I will rework this to make it a bit less robotic.